### ISSUE TITLE
The results of GNN method

### ISSUE THREAD

**Issue by nuistji** [Haoqing-Wang/CDFSL-ATA#1]
Nice work! However, I cannot reimplement the result in table 1 using the GNN method. I notice that your finetune.py document also covers the augmentation method. I finetuned the GNN checkpoint which was trained by step 1 and get very low accuracy using the finetune.py from https://github.com/IBM/cdfsl-benchmark. So can you provide me some detailes on how to get the GNN results in table 1. Also, I noticed that your code can not cover more shots(5way-20s or 5w-50s), since this may cost more CUDA memerory, have you solved this issue?

**Comment by MichalisLazarou** [comment]
Hey I also have this issue, finetuning does not seem to improve the performance!

**Comment by Haoqing-Wang** [comment]
Hi, I advise to fine-tune the models using my code. Concretely, if you want to fine-tune the meta-learning models, please use https://github.com/Haoqing-Wang/CDFSL-ATA/blob/master/finetune_ml.py, if you want to fine-tune a linear classifier, please use https://github.com/Haoqing-Wang/CDFSL-ATA/blob/master/finetune.py

**Comment by MichalisLazarou** [comment]
Do you only fine-tune the last layer or the whole embedding model? because i tried both your codes finetune.py and finetune_ml.py and it seems the results are getting worse with finetuning especially at 1-shot

**Comment by Haoqing-Wang** [comment]
I fine-tune all the layers.
